docs/API.md

```diff
@@ -955,42 +955,38 @@ __Parameters__
 | Param  |  Type | Description  |
 |---|---|---|
 | `bucketName`  | _string_  | Name of the bucket |
-| `objectPrefix` | _string_ | Prefix of objects in the bucket with which to filter permissions off of. Use `''` for entire bucket. |
-| `callback(err, policy)`  | _function_  | Callback function is called with non `null` err value in case of error. `policy` will be the string representation of the bucket policy (`minio.Policy.NONE`, `minio.Policy.READONLY`, `minio.Policy.WRITEONLY`, or `minio.Policy.READWRITE`). If no callback is passed, a `Promise` is returned. |
+| `callback(err, policy)`  | _function_  | Callback function is called with non `null` err value in case of error. `policy` will be the string representation of the bucket policy file. If no callback is passed, a `Promise` is returned. |
 
 
 ```js
 // Retrieve bucket policy of 'my-bucketname' that applies to all objects that
 // start with 'img-'.
-minioClient.getBucketPolicy('my-bucketname', 'img-', function(err, policy) {
+minioClient.getBucketPolicy('my-bucketname', function(err, policy) {
   if (err) throw err
 
-  console.log(`Bucket policy: ${policy}`)
+  console.log(`Bucket policy file: ${policy}`)
 })
 ```
 
 <a name="setBucketPolicy"></a>
-### setBucketPolicy(bucketName, objectPrefix, bucketPolicy[, callback])
+### setBucketPolicy(bucketName, bucketPolicy[, callback])
 
-Set the bucket policy associated with the specified bucket. If `objectPrefix`
-is not empty, the bucket policy will only be assigned to objects that fit the
-given prefix.
+Set the bucket policy associated with the specified bucket.
```

Comment:
May be:
`Set the bucket policy associated with the specified bucket.`
 =>
`Set the bucket policy/policies provided in JSON file, bucketPolicy, associated with the specified bucket.`